omod/src/test/resources/TagResourceDataSet.xml

```diff
@@ -0,0 +1,29 @@
+<dataset>
```

Comment:
Why don't you use the same test dataset that you are using for the java api tests?